packaging/python_wheels/mac/build.sh

```diff
@@ -0,0 +1,286 @@
+#!/bin/bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+function realpath
+{
+    python -c "import os; print(os.path.realpath('$1'))"
+}
+
+WORKSPACE=$(realpath `pwd`/../../../..)
+SOURCE_PATH=$WORKSPACE/Brion
+
+# Cannot use an earlier version because C++11 features don't compile otherwise
+export MACOSX_DEPLOYMENT_TARGET="10.9"
+export PYTHON_CONFIGURE_OPTS="--enable-universalsdk=/ --with-universal-archs=intel"
+export PYENV_ROOT=$WORKSPACE/pyenv
+export PATH="$PYENV_ROOT/bin:$PATH"
+export HTTP_PROXY=bbpproxy.epfl.ch:80
```

Comment:
this is already setup'd by CI, really needed here?

Reply:
If my memory doesn't fail, I remember having network errors, which is why I added them. The build_wheel.sh script for Linux does this:

    export http_proxy=${HTTP_PROXY-$http_proxy}
    export https_proxy=${HTTPS_PROXY-$https_proxy}

So I will replace these lines with that.